kivymd/uix/button/button.py

```diff
@@ -1059,6 +1059,11 @@ class ButtonContentsIconText:
     and defaults to [12dp, 8dp, 16dp, 8dp].
     """
 
+    icon_position = OptionProperty("left", options=["left", "top"])
+    """
```

Comment:
Please follow the specification of the KivyMD library, which regulates how to create doc strings for class attributes.

Reply:
I looked at the other parts of the code and I hope the doc strings are now in the right format.